EIPS/eip-823.md

```diff
@@ -187,13 +187,13 @@ event ExchangeAndSpent ( address _from, address _by, uint _value ,address _targe
 ### Diagramatic Explaination
 
 #### Exchanging Tokens
-![token-exchange-standard-visual-representation-1](https://raw.githubusercontent.com/kashishkhullar/EIPs/master/EIPS/token-exchange-standard-visual-representation-1.PNG)
+![token-exchange-standard-visual-representation-1](https://raw.githubusercontent.com/kashishkhullar/EIPs/master/assets/eip-823/eip-823-token-exchange-standard-visual-representation-1.PNG)
 
 NOTE: After the successful exchange the contract on right owns some tokens of the contract on the left.
 
 #### Exchanging And Spending Tokens
 
-![token-exchange-standard-visual-representation-2](https://raw.githubusercontent.com/kashishkhullar/EIPs/master/EIPS/token-exchange-standard-visual-representation-2.PNG)
+![token-exchange-standard-visual-representation-2](https://raw.githubusercontent.com/kashishkhullar/EIPs/master/assets/eip-823/eip-823-token-exchange-standard-visual-representation-2.PNG)
```

Comment:
@kashishkhullar you have added the files to the `assets` directory, but these links are pointing to a remote repo. Can you change these links to be `.. /assets/eip-823/eip-823-token-exchange-standard-visual-representation-2.PNG` ?

Reply:
I have updated the links.

Reply:
Thanks!